dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/SymbolProblem.java

```diff
@@ -97,13 +97,16 @@ public static String formatSymbolProblems(
       ImmutableSetMultimap<SymbolProblem, ClassFile> symbolProblems) {
     StringBuilder output = new StringBuilder();
 
-    for (SymbolProblem problem : symbolProblems.keySet()) {
-      int referenceCount = symbolProblems.get(problem).size();
-      output.append(
-          String.format(
-              "%s\n  referenced by %d class file%s\n",
-              problem, referenceCount, referenceCount > 1 ? "s" : ""));
-    }
+    symbolProblems
+        .asMap()
+        .forEach(
+            (problem, classFiles) -> {
+              int referenceCount = classFiles.size();
```

Comment:
By using `asMap.forEach`, no need to call `get`. (https://github.com/GoogleCloudPlatform/cloud-opensource-java/pull/629/files#r288151047)